javaslang/src/main/java/javaslang/collection/Multimap.java

```diff
@@ -233,6 +233,15 @@ default void forEach(BiConsumer<K, V> action) {
     Option<Traversable<V>> get(K key);
 
     /**
+     * Returns the value associated with a key, or a default value if the key is not contained in the map
+     *
+     * @param key the key
+     * @param defaultValue a default value
+     * @return the value associated with key if it exists, otherwise the result of the default value
+     */
+    Traversable<V> getOrElse(K key, Traversable<V> defaultValue);
```

Comment:
Can we widen the defaultValue type to `Traversable<? extends V>` without breaking anything?